### ISSUE TITLE
Allow new users to submit tickets to Zammad

### ISSUE THREAD

**Issue by sonjafeitsch** [sonjafeitsch/meine-nische-booking#2]
**Requirements:** The booking tool for our event and yoga space creates a new event in a calender and a new ticket in [Zammad](https://zammad.com/en) (a ticketing system). Due to Zammads API Design a user should exist before we can create a new ticket via the corresponding endpoint. Therefore our booking tool works only for registrated users in our ticket system. For the future it would be nice if users that does not exist in our system can reserve their time slots.
